src/Image.php

```diff
@@ -355,7 +355,7 @@ private function getPixelValue($size)
         $size = trim($size);
 
         if (isset($map[substr($size, -2)])) {
-            $size = $size * $map[substr($size, -2)];
+            $size = substr($size, 0, -2) * $map[substr($size, -2)];
```

Comment:
You are right, but we also have to check if the result is a numeric string IMO. I changed that in b45a55ee0261041eb92086fbebe99a8731be66b0.